CMakeLists.txt

```diff
@@ -23,14 +23,32 @@ endif()
 
 enable_testing()
 
+# populate submodule blazesym
+if(NOT NO_BLAZESYM)
+    execute_process(COMMAND git config --global --add safe.directory ${CMAKE_CURRENT_SOURCE_DIR}
```

Comment:
This `execute_process` statement and 

> Failed to add root source directory to safe.directory"

warning check should happen for both blazesym and libbpf submodule usage. I think it's fine for it to live outside of both `if(NOT NO_BLAZESYM)` and `if(NOT CMAKE_USE_LIBBPF_PACKAGE)` checks.